Tests/FluentAssertions.Specs/Equivalency/ExtensibilityRelatedEquivalencySpecs.cs

```diff
@@ -154,6 +154,50 @@ public SelectedMemberInfo Match(SelectedMemberInfo expectedMember, object subjec
 
         #endregion
 
+        #region Ordering Rules
+
+        [Fact]
+        public void When_an_ordering_rule_is_added_it_should_be_evaluated_after_all_existing_rules()
+        {
+            // Arrange
+            var subject = new[] { "First", "Second" };
+            var expected = new[] { "First", "Second" };
+
+            // Act
+            Action act = () => subject.Should().BeEquivalentTo(
+                expected,
+                options => options.Using(new StrictOrderingRule()));
+
+            // Assert
+            act.Should().NotThrow();
+        }
+
+        [Fact]
+        public void When_an_ordering_rule_is_added_it_should_appear_in_the_exception_message()
+        {
+            // Arrange
+            var subject = new[] { "First", "Second" };
+            var expected = new[] { "Second", "First" };
+
+            // Act
+            Action act = () => subject.Should().BeEquivalentTo(
+                expected,
+                options => options.Using(new StrictOrderingRule()));
+
+            act.Should().Throw<XunitException>()
+                .WithMessage(string.Format("*{0}*", typeof(StrictOrderingRule).Name));
```

Comment:
Why don't you use $"*{typeof(StrictOrderingRule).Name}*"?

Reply:
I sticked to the way it's been done in the other tests in the same file for consistency. But I'd happily adjust it, maybe for all places in this file for consistency?
I was also thinking to use `nameof`, though I'm not sure this works with all supported frameworks.

Reply:
> Maybe for all places in this file for consistency?

Only if you feel like it.

> I was also thinking to use nameof, though I'm not sure this works with all supported frameworks.

If it compiles, it most likely runs.

Reply:
`nameof` expressions is a C# 6 language construct independent of the runtime.
Once compiled it's just a string.

Reply:
done consistently in the whole file